feature-libs/checkout/base/occ/adapters/converters/occ-checkout-normalizer.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { CheckoutState } from '@spartacus/checkout/base/root';
```

Comment:
Do we really need this? We're not manipulating anything coming from the backend's response, so we can safely remove it. The customer can always map the token to there custom normalizer.

BUT!, if we do keep it, then we'd need to create the other normalizers as well :P

Reply:
You are right - removed it.